platform/pensando/sonic-platform-modules-dpu/dpu/utils/dhcp_reset.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Define the interface
+INTERFACE="int_mnic0"
+
+# Delete any preexisting IP addresses on the interface
+ip addr flush dev $INTERFACE
+
+dhclient $INTERFACE
+# Infinite loop to ping every 10 seconds
+while true; do
+    dhclient $INTERFACE
+    sleep 180
+done
```

Comment:
@shanshri who else is configuring this IP other the than the sonic dhcp service that needs this to be cleaned up?

Reply:
Our Polaris fw container configures int_mnic0 static IP with PCIE ID, that need to be cleaned up and need to send fresh dhcp request.